src/Components/Results/Results.css

```diff
@@ -283,6 +315,7 @@
     width: 40%;
     vertical-align: top;
     line-height: normal;
+    color: #293457;
```

Comment:
We define the color on line 136 for this `.result-program-more-info`, so we don't need to redefine it for this media query.
```suggestion
```